sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md

```diff
@@ -0,0 +1,77 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: Qumulo
+namespace: Azure.ResourceManager.Qumulo
+require: https://github.com/Azure/azure-rest-api-specs/blob/aceee26b40a3251a1dbcc76e57fb1e0860f1c76d/specification/liftrqumulo/resource-manager/readme.md
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+
```

Comment:
Please add a `generate-model-factory: false` here to exclude the model factory.
The model factories are on our plan but not quite ready to release yet

Reply:
done